### PR TITLE
msg

### PR DESCRIPTION
**Pull request by santoshinawkhare**

### CODE REVIEW

**Review by Manishnerkar — APPROVED** [review]

**Review by pradip0852 — APPROVED** [review]

File: Assignments/HTML_CSS/Assign1/santoshiA1/assignment A1.2.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    <head><h1>HTML</h1></head>
+    <body>
+        <h4>Introduction</h4>
+        <p>HTML is a standard markup language for Web Pages</p>
+        <h5>HTML Editor</h5>
+        <h3>Notepad or TextEditor.</h3>
+        <p>Web Pages can be created and modified by using Professional HTML editor.</p>
+        <p>However, for lerning HTML. we recommend a simple text editor like Notepad (PC) or TextEdit (Mac.)</p> 
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/santoshiA1/assignment A1.2.html:11]: add new line

File: Assignments/HTML_CSS/Assign1/santoshiA1/assignmentA1.1html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html>
+    <head>
+        <h1>HTML</h1>
+    </head>
+    <body>
+        <h4><b>Introduction</b></h4>
+        <p>HTML ia a standard markup language for Web Pages.</p>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/santoshiA1/assignmentA1.1html:9]: add new line

File: Assignments/HTML_CSS/Assign1/santoshiA1/assignment A1.2.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<html>
+    <head><h1>HTML</h1></head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/santoshiA1/assignment A1.2.html:2]: need title here

File: Assignments/HTML_CSS/Assign1/santoshiA1/assignmentA1.1html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<html>
+    <head>
+        <h1>HTML</h1>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/santoshiA1/assignmentA1.1html:3]: need title here

File: Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<html>
+    <head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html:2]: head tag closing

File: Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        <p><i><b>Larry Page</b></i> was considering standford for grad school and <i><b>Sergey Brin</b></i>, a student there, was assigned to show him around.</p>
+        <p style="font-size: small;">By some account, they disagreed about nearly everything during that first meeting, but the following year they struck a <mark>partnership</mark>.</p>
+        <p style="font-size: small;"> Working from their dorm room, they build a search engin tat used links to determine the importance of individual pages on the <b style="color:rgb(36, 103, 228);">Word Wide Web</b></p><br>
+        <p style="font-size: small;">They called this search engin <i><b style="background-color: rgb(92, 179, 238);">Backrub</b></i></p>
+        <hr>
+        <h2>Initial public offering</h2>
+        <p style="font-size: small;">Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004.<br>
+        At that time Larry page, Sergey Brin, and Erin Schmidt agreed to work together at Google for <del>30</del><u>20</u><br>Years, until the year 2024.</p>
+        <h3><p style="color:darkgreen"><marquee behavior="alternate" >Thank You!!</marquee></p></h3>
+        </head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html:17]: remove head tag form here

File: Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+         </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/santoshiA2/assignment2.html:19]: add new line

File: Assignments/HTML_CSS/Assign3/santoshiA3/AssignmentA3.1.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
+<html>
+    <head>
+      <title>lists</title>
+      <style>
+        ul{color:brown; list-style: disc;}
+        a{color: brown;}
+        ul a1{color:blue;}
+        a1{color:blue;}
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/santoshiA3/AssignmentA3.1.html:8]: wrong css

File: Assignments/HTML_CSS/Assign6/santoshiA6.html [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+    <fieldset>
+        <legend>Resume</legend>
+        <label for="Resume">Upload Resume:&nbsp;</label>
+        <input type="Resume" id="Resume" name="Resume" placeholder="Browose....">&nbsp;&nbsp;No such file select.
+        <br></br>
+        <label for="Resume">comments:&nbsp;&nbsp;</label>
+        <textarea name="comments" id="Comments" cols="30" rows="10"></textarea>
+    </fieldset>
+    <br></br>
+       <centre><button>style="text-align: center;">Submit</button></centre>>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/santoshiA6.html:93]: remove extra  > sign

File: Assignments/HTML_CSS/Assign6/santoshiA6.html [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+    <br>
+    <fieldset>
+        <legend>Resume</legend>
+        <label for="Resume">Upload Resume:&nbsp;</label>
+        <input type="Resume" id="Resume" name="Resume" placeholder="Browose....">&nbsp;&nbsp;No such file select.
+        <br></br>
+        <label for="Resume">comments:&nbsp;&nbsp;</label>
+        <textarea name="comments" id="Comments" cols="30" rows="10"></textarea>
+    </fieldset>
+    <br></br>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/santoshiA6.html:92]: no need of closing tag

File: Assignments/HTML_CSS/Assign6/santoshiA6.html [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+        <option value="mgm">management</option>
+    </select>
+    You can select Multiple field.
+    </fieldset>
+    <br>
+    <fieldset>
+        <legend>Resume</legend>
+        <label for="Resume">Upload Resume:&nbsp;</label>
+        <input type="Resume" id="Resume" name="Resume" placeholder="Browose....">&nbsp;&nbsp;No such file select.
+        <br></br>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/santoshiA6.html]: no need of closing tag

File: Assignments/HTML_CSS/Assign6/santoshiA6.html [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+    <br></br>
+    <lable for="Qualification">Highest Qualification:</lable>
+    <select id="Qualification" name="Qualification">
+        <option value="MI-9">Diploma in Computer Science</option>
+        <option value="MI-8">Sciences Graduate</option>
+        <option value="MI-7">Bachelor in engg.</option>
+        <option value="MI-6">Master in Computer Science</option>
+        <option value="MI-6">Master in Engg.</option>
+        <option value="MI-6">Other</option>
+    </select><br></br>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/santoshiA6.html:70]: no need of closing tag

File: Assignments/HTML_CSS/Assign6/santoshiA6.html [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+        <label for="university">University:&nbsp;</label>
+        <input type="university" id="university" name="university">
+        <br></br>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/santoshiA6.html]: same here

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request merged**